narwhal/source_code_sharing/update_liferay_dxp_source_code_of_hotfixes.sh

```diff
@@ -32,19 +37,39 @@ function check_if_tag_exists {
 	fi
 }
 
+function check_ignore_argument {
+	if [[ "x${IGNORE_ZIP_FILES}" =~ x*${hotfix_zip_file}* ]]
+	then
+		lc_log DEBUG "Ignoring '${file_url}'."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_OK}"
+	else
+		lc_log DEBUG "The file on '${file_url}' is not on the ignore list."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_BAD}"
+	fi
+}
+
 function check_ignore_zip_file {
 	local hotfix_zip_file="${1}"
-	local release_version="${2}"
 
 	local file_url="${zip_directory_url}/${hotfix_zip_file}"
 
-	if [[ "x${IGNORE_ZIP_FILES}" =~ x*${hotfix_zip_file}* ]]
+	check_ignore_argument 
+	check_ignore_txt_file "${IGNORE_ZIP_FILES_PRESISTENT_FILE}"
+	check_ignore_txt_file "${IGNORE_ZIP_FILES_EXPANDING_FILE}"
+}
+
+function check_ignore_txt_file {
+	local ignore_zip_file="${1}"
+
+	if [[ $(grep -q "${hotfix_zip_file}" "${ignore_zip_file}") ]]
```

Comment:
No [[ $(command)]] is needed. Actually in this case it should not even work. It should be just `if (command)`